docs/src/main/asciidoc/transaction-guide.adoc

```diff
@@ -113,6 +113,13 @@ public class SantaClausService {
 <1> Inject the `TransactionManager` to be able to activate `setRollbackOnly` semantic.
 <2> Programmatically decide to set the transaction for rollback.
 
+
+=== Transaction Configuration
+Extended configuration of the transaction is possible with the use of the `@TransactionConfiguration` annotation that needs to be set in addition to the standard `@Transactionnal` annotation (at method or class level).
+
+The `@TransactionConfiguration` allow to set a `timeout` property, in seconds, that applies to the current transaction.
```

Comment:
You probably need to mention that this annotation will only take effect if you place it on the top level method delineating the transaction and that annotated nested methods once a transaction has started will be a noop (or a warning I forgot).

Reply:
+1 (or even indeed it might be an exception if that is what is decided below)

Reply:
I rephrase it with the notion of entry method as it is already conveyed in the transactional guide.

Reply:
Well, it is not changing the current transaction. So maybe something like:

> The `@TransactionConfiguration` allow to set a `timeout` property, in seconds, that applies to transactions created within the annotated method.

Reply:
done